src/main/java/com.company/heap/NMaxPairCombination.java

```diff
@@ -0,0 +1,46 @@
+package com.company.heap;
+
+import java.util.*;
+
+public class NMaxPairCombination {
```

Comment:
If you have understood the problem, there's a problem with a small variation in Leetcode.
https://leetcode.com/problems/find-k-pairs-with-smallest-sums/
See if you can solve this